corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -73,7 +73,8 @@ def handle(self, existing_domain, new_domain, **options):
             self.copy_fixtures()
 
         if self._clone_type(options, 'locations'):
-            self.copy_locations()
+            types_only = self._clone_type(options, 'types_only')
```

Comment:
this option breaks the patter of include / exclude options. The alternative of splitting locations and location types completely is more complex though since there is a dependency between the two.

I'm torn between the two options but I think let's stick with this to keep it simple.